chromadb/__init__.py

```diff
@@ -165,6 +173,14 @@ def HttpClient(
     if settings is None:
         settings = Settings()
 
+    # https://github.com/chroma-core/chroma/issues/1573
+    host = str(host)
+    port = int(port)
+    ssl = bool(ssl)
+    headers = _stringify_headers(headers)
```

Comment:
should we insist all headers are strings?

Reply:
We do this with the type annotations.

However type annotations are statically typed only when you feel like it and don't actually enforce anything at runtime -- this PR exists because a user was hitting a subtle bug when they passed in an `int` for `port` instead of a `string`.

Our options here are to stringify headers or simply pass them as they're passed to us. @tazarov requested below that we pass them as-is and I don't feel strongly about it so that's what we do now. Removed the `_stringify_headers` stuff.